ui-v2/app/styles/components/table.scss

```diff
@@ -2,6 +2,7 @@
 @import './tag';
 td strong {
   @extend %tag;
+  background-color: $ui-gray-100;
```

Comment:
I'm just wondering why I haven't put a color for 'tags' some centralized already? I think this should go somewhere centralized for the project but not centralized in the `%tag` definition as different projects might what different colors, maybe we add a 'default' in `%tag`, and it can just be overridden?

Will have a better look in the code once I get going in a sec.

Reply:
P.S. Just to note here just incase, `%tag` is getting moved to `%pill` at some point soon.